frontend/src/scenes/feature-flags/FeatureFlagInstructions.tsx

```diff
@@ -299,44 +190,86 @@ export function CodeInstructions({
                             }}
                             value={selectedOption.value}
                         />
-                        {PAYLOAD_OPTIONS.map((payloadOption) => payloadOption.value).includes(selectedOption.value) && (
+                        <div className="flex items-center gap-1">
+                            <LemonCheckbox
+                                label="Show payload option"
+                                onChange={() => setShowPayloadCode(!showPayloadCode)}
+                                checked={showPayloadCode}
+                                disabled={
+                                    !PAYLOAD_OPTIONS.map((payloadOption) => payloadOption.value).includes(
+                                        selectedOption.value
+                                    )
+                                }
+                            />
+                            <Tooltip
+                                title={
+                                    <>
+                                        {`Feature flag payloads is only available in these libraries: ${PAYLOAD_OPTIONS.map(
```

Comment:
```suggestion
                                        {`Feature flag payloads are only available in these libraries: ${PAYLOAD_OPTIONS.map(
```

Reply:
Payloads belong to the feature flag which is a singular noun here so it should be `is` 

If I wrote just payloads then it should be `are`

Reply:
Huuuh, interesting. Still sounds wrong to me lol. @joethreepwood do you have any suggestions here, or is it just me who's weird? :P

Reply:
happy to change to what is "correct" but also this is the type of pending PR feedback that's not really useful or productive and borders micro manage-y nitpick 🙊

Reply:
Hmm, I see it slightly differently, where attention to this kind of detail on the frontend makes a difference to the UX. That's because this is what users will see, and I instantly lose trust in a product that can't get words right, makes me feel like "not sure they pay attention to things".

Reply:
I agree attention to detail is a must, but contextually this is a PR to continue the work of the code example feat and not a final rendition of things, right? 

I appreciate the detailed feedback, some of it just feels inappropriately placed if we're discussing the "is vs are" of a sentence when you can see we're still working through other bigger things. 

I know you have good intentions but the feedback comes across as microscopic and argumentative especially if I go "hey some of this feedback is feeling nitpicky" and you go "_no_, because I see XYZ differently" which just disregards what I'm saying lol

Reply:
That's fair, thanks for the feedback! Also agree on my previous message seeming like it disregards yours.

Not necessary to answer this if it takes too much time, but do you have any recommendations on how I could've responded instead of "Hmm, I see it slightly differently" ? 😅

Reply:
The more collaborative response here would be to try to understand why I might've felt that way if you disagreed with my "it's giving nitpick" feedback and you saw it differently and to listen first instead of to respond. 

And to the initial comment of where I literally explained why it's `is` instead of `are`, your response of `Huuuh, interesting. Still sounds wrong to me lol.` is not helpful (and kind of a bit rude) because I provided you with a reasonable and factual explanation of my argument and you hit me with the "but my opinion says that sounds wrong" 🙈

Reply:
Actually, on reflection I think I figured out what I should've done here instead!

which is to ask you why you felt this was nitpicky, to which you would've replied __"this is a PR to continue the work of the code example feat and not a final rendition of things, right?"__ which wasn't clear to me, but asking this would've made it clear.

Heh, makes a lot of sense. Do you agree on this being a better way around it, or nah?

Reply:
> is not helpful (and kind of a bit rude) because I provided you with a reasonable and factual explanation of my argument and you hit me with the "but my opinion says that sounds wrong" 🙈

Thanks for taking the time to respond, very helpful! Agree, could've framed this much better!

Reply:
`are` is correct. 

In this context, `feature flag payloads` is a single (but not singular) noun. It's all one thing (e.g. `a feature flag payload`), and it's discussed as a multiple, so it's `are`. 

I definitely wouldn't sweat things like this in terms of UI. I can always go through and do a sweep or add a review whenever you like -- and I think we're all pretty good at fixing things when we catch them.

Reply:
Much appreciate the special ops once the feature is out, that's a great idea!

Reply:
FYI I was also just using this while trying to figure it out 😎 https://www.grammarly.com/blog/is-vs-are/?psafe_param=1&gclid=Cj0KCQjwtsCgBhDEARIsAE7RYh21eOHJLPHlDgwOsm5YTb29jEh5LAXo-ymqaP3-EaTz_uVa7e3AdjsaAhb0EALw_wcB&gclsrc=aw.ds just in case you were curious @neilkakkar